examples/react-testing-lib/src/components/input.test.tsx

```diff
@@ -56,3 +58,34 @@ describe('Input', async () => {
     expect(screen.getByRole('alert')).toHaveTextContent('Please enter your email')
   })
 })
+
+describe('ui console view', () => {
+  // https://github.com/vitest-dev/vitest/issues/2765
+  it('regexp', () => {
+    console.log('A:', /(?<char>\w)/)
+    console.log('B:', /(?<char>\w)/.source)
+    console.log(`C: ${/(?<char>\w)/}`)
+  })
+
+  // https://github.com/vitest-dev/vitest/issues/3934
+  it('html', async () => {
+    const fs = await import('node:fs')
+    const file = await fs.promises.readFile('src/components/Input.tsx', 'utf-8')
+    console.log(file)
+  })
+
+  // https://github.com/vitest-dev/vitest/issues/1279
+  it('screen.debug', () => {
+    render(
+      <Input
+        name="email"
+        type="email"
+        placeholder="Email"
+        label="Email Address"
+        aria-label="Email Address"
+        error="Please enter your email"
+      />,
+    )
+    screen.debug()
+  })
+})
```

Comment:
I confirmed these cases mentioned in the issues are now looking okay:

<details><summary>Reveal screenshot</summary>

![image](https://github.com/vitest-dev/vitest/assets/4232207/9eb750f5-39ee-42e8-bddd-49979e0198e7)

</details>

Probably it's a little off-putting to have these logging in this example, so I plan to move it to elsewhere.
Please let me know if you have any preference.

Reply:
For now I removed these test cases https://github.com/vitest-dev/vitest/pull/4724/commits/26ea82a0759bdae6e477b844ce2fa6110785c277